notion/client.py

```diff
@@ -373,7 +374,46 @@ def create_record(self, table, parent, **kwargs):
                 )
 
         return record_id
+    
+    def get_task_status(self, task_id):
+        """
+        Get a status of a single task
+        """
+        data = self.post(
+            "getTasks",
+            {
+                "taskIds": [task_id]
+            }
+        ).json()
+
+        results = data.get("results")
+        if results is None:
+            return None
+        
+        if not results:
+            # Notion does not know about such a task
+            print("Invalid task ID.")
+            return None
 
+        if len(results) == 1:
+            state = results[0].get("state")
+            return state
+        
+        return None
+        
+    def wait_for_task(self, task_id, interval=1, tries=5):
```

Comment:
Would this be long enough for a huge template doc (with lots of children)? Worth upping the tries to 10, do you think?

Reply:
done.
We can also consider passing `interval` and `tries` as an optional variable to `add_row(...)` so it can be configured when calling the function that uses `wait_for_task(...)` and not only when directly calling `wait_for_task(...)`.

So currently, users can't control the intervals and tries of `add_row(...)` and I am not sure they should